source/reference/hyperlinks.txt

```diff
@@ -0,0 +1,102 @@
+.. _style-guide-hyperlinks:
+
+==========
+Hyperlinks
+==========
+
+You can link out to other sites, internal pages or specific sections on
+a page using inline hyperlinks and cross-referenced hyperlinks.
```

Comment:
I think you need a serial comma before the "or". I even found a citation. :)

https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#commas